vimeo-networking/src/main/java/com/vimeo/networking/callbacks/IgnoreResponseVimeoCallback.java

```diff
@@ -22,24 +24,18 @@
 
 package com.vimeo.networking.callbacks;
 
-import org.jetbrains.annotations.NotNull;
-
 /**
- * Abstract callback class for {@code T} type of model.
+ * Use this callback when you don't care about the server response and therefore don't need to deal
+ * with typed network calls.
  * <p>
- * Created by hanssena on 4/27/15.
+ * Created by rigbergh on 7/20/17.
  */
-public abstract class ModelCallback<T> extends VimeoCallback<T> {
+public abstract class IgnoreResponseVimeoCallback extends VimeoCallback<Object> {
```

Comment:
Good question. Primarily it solves a practical problem with the put, post and delete calls in VimeoClient. In our codebase, we are ignoring the response object in all of uses of the API when onSuccess is called. In the instances where we need an object, we are referencing an in-memory version of the object that is established before the networking request is made rather than using the object returned from the server (which would be a better practice). The alternative would be to blow out typed versions of put, post and delete calls (as I did for the get requests) when our app doesn't actually need them. It reduced the scope of the problem dramatically. I don't think that this is a great long-term solution for the networking API, but it solves the problem for now. After this PR is merged, I am going to create a series of tickets regarding TBD issues that we need to address, and this is one of them. Some of this overlaps with the object tracking initiative, so we will return to this issue one way or another.